extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -187,4 +187,12 @@ public static final FlywayDataSourceRuntimeConfig defaultConfig() {
      */
     @ConfigItem
     public boolean validateMigrationNaming;
+
+    /**
+     * Ignore migrations during validate and repair according to a given list of patterns,
```

Comment:
It isn't totally clear what patterns are these, maybe add a link to https://flywaydb.org/documentation/configuration/parameters/ignoreMigrationPatterns here or include the pattern format from that page here?